packages/esbuild/esbuild.bzl

```diff
@@ -308,10 +337,11 @@ def esbuild_macro(name, output_dir = False, **kwargs):
         entry_point = Label("@build_bazel_rules_nodejs//packages/esbuild:launcher.js"),
     )
 
-    if output_dir == True:
```

Comment:
This will cause an error when `output_dir` is `True` as it will pass `entry_points = False`, where `entry_points` is expecting a label list, but will get a boolean.

Perhaps something like the following (`None` is falsy and the default on `label_list` is an empty list)
```
entry_points = kwargs.get("entry_points", None)
if output_dir == True or entry_points:
    esbuild(
            name = name,
            output_dir = True,
            launcher = _launcher,
            **kwargs
        )
```

It can then be removed from the macro's signature